sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/api/Azure.ResourceManager.DataProtectionBackup.netstandard2.0.cs

```diff
@@ -900,6 +922,31 @@ public partial class CopyOnExpirySetting : Azure.ResourceManager.DataProtectionB
     {
         public CopyOnExpirySetting() { }
     }
+    public partial class CrossRegionRestoreDetails
+    {
+        public CrossRegionRestoreDetails(string sourceRegion, string sourceBackupInstanceId) { }
+        public string SourceBackupInstanceId { get { throw null; } }
+        public string SourceRegion { get { throw null; } }
+    }
+    public partial class CrossRegionRestoreJobContent
+    {
+        public CrossRegionRestoreJobContent(string sourceRegion, string sourceBackupVaultId, string jobId) { }
+        public string JobId { get { throw null; } }
+        public string SourceBackupVaultId { get { throw null; } }
```

Comment:
same here, are these ids `arm resource id`? If so please change their format to `arm-id` accordingly.